### DISCUSSION TITLE
Please add captcha form

### DISCUSSION THREAD

**Discussion by jet10000** [jrief/django-formset#22] [Ideas]
https://github.com/J-Rios/multicolorcaptcha

Are there plans to add captcha form? or other solutions.

**Comment by jrief** [comment]
never thought of this, but it's a neat idea.

**Comment by jet10000** [comment]
+1

**Comment by jrief** [comment]
I probably will never add a Captcha field to django-formset. See also #53 .

1. Reason is that AI/OCR nowadays are even better than humans in solving them.
2. Reason is that they are the opposite of good UX.

I might add some kind of proof-of-work in order to prevent brute-force attacks on forms.
